additional files

robsalomone opened the pull request:
Included the Phase Transition example files, and the supplementary material pdf.

LeahPrice commented:
Thanks Rob for adding this. It's good to have it all in one place.

There were a lot of functions that were exact duplicates of what's in the code folder, so I added the code folder to the path and deleted them. The only one I haven't deleted is bisection because you have a tolerance of 10e-7 and I had 10e-6. Are you okay with me making them both the same, maybe 10e-6? If you're happy with this then we can delete the bisection function and merge your changes.

robsalomone commented:
Hi Leah, sure go ahead. Just keep the old version on a zip file in Dropbox just in case. 



> On 16 Aug 2018, at 11:37 am, Leah South <notifications@github.com> wrote:
> 
> Thanks Rob for adding this. It's good to have it all in one place.
> 
> There were a lot of functions that were exact duplicates of what's in the code folder, so I added the code folder to the path and deleted them. The only one I haven't deleted is bisection because you have a tolerance of 10e-7 and I had 10e-6. Are you okay with me making them both the same, maybe 10e-6? If you're happy with this then we can delete the bisection function and merge your changes.
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub, or mute the thread.

LeahPrice commented:
Great, thanks. I've removed the duplicate now. Fortunately Github keeps track of all of our old files so the zip file shouldn't be necessary.
